.github/workflows/ci.yml

```diff
@@ -40,7 +40,9 @@ jobs:
           with:
             go-version: 1.17.x
         - name: Install Ko
-          uses: imjasonh/setup-ko@20b7695b536c640edfafdd378d96c760460f29d6
+          uses: imjasonh/setup-ko@v0.4
```

Comment:
supernit: At some point folks wanted these to be specified by commit, in which case `v0.4` is `2c3450ca27f6e6f2b02e72a40f2163c281a1f675` -- I'm not sure whether it matters (I pinky promise not to change the v0.4 tag).

Reply:
I'm fine doing the sha as well - "zero trust" am I right?

Reply:
Yep, +1.